src/components/BackButton.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <RouterLink :to="props.prevPage">
+    <div class="text-gray_05-light dark:text-gray_05-dark w-6 h-6">뒤</div>
+  </RouterLink>
+</template>
+
+<script setup lang="ts">
+import { RouterLink } from 'vue-router';
+
+const props = defineProps<{
+  prevPage: string;
+}>();
```

Comment:
`const props =` 할당은 script 단에서 props를 사용할 경우에만 할당해주시면 됩니다.
지금처럼 template 내부에서만 사용할 경우에는 아래와 같이 사용해주시면 됩니다.
```suggestion
<RouterLink :to="prevPage"> //template 내부에서는 props 생략


defineProps<{
  prevPage: string;
}>();
```